.github/workflows/rw-backup-maria-database.yml

```diff
@@ -0,0 +1,71 @@
+name: RW — Backup MariaDB 
+
+on:
+  workflow_call:
+    inputs:
+      github_env:
+        type: string
+
+      database_host:
+        type: string
+        required: true
+
+      database_name:
+        type: string
+        required: true
+
+      database_user:
+        type: string
+        required: true
+
+      storage_account_name:
+        type: string
+        required: true
+
+      storage_container_name:
+        type: string
+        required: true
+      
+      backup_file_name:
+        type: string
+        required: true
+
+    secrets:
+      database_password:
+        required: true
+      
+      azure_login_creds:
+        required: true
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  backup:
+    name: 'Backup MariaDatabase'
+    runs-on: ubuntu-latest
+
+    environment:
+      name: ${{ inputs.github_env  }}
+
+    steps:
+
+      - name: Create
+        run: |
+          MYSQL_PWD='${{ secrets.database_password }}' mysqldump --column-statistics=0 --force --opt --skip-lock-tables --host=${{ inputs.database_host }} --user=${{ inputs.database_user }} cca_quote_dev | gzip > ${{ inputs.backup_file_name }}
```

Comment:
```suggestion
          MYSQL_PWD='${{ secrets.database_password }}' mysqldump --column-statistics=0 --force --opt --skip-lock-tables --host=${{ inputs.database_host }} --user=${{ inputs.database_user }} ${{ inputs.database_name }} | gzip > mariadb_backup.bak
```

Reply:
This is done because we're creating a fixed backup file which is gzipped. We want the other pipeline to pick up this file and upload to azure, but give azure a new file name to use, which is `backup_filename` as defined above